packages/block-editor/src/components/block-navigation/README.md

```diff
@@ -0,0 +1,34 @@
+# Block navigation
+
+The BlockNavigation component provides an overview of the hierarchical structure of all blocks in the editor. The blocks are presented vertically one below the other.
+
+Blocks that have child blocks (such as group or column blocks) are presented with the parent at the top and the nested children below.
+
+In addition to presenting the structure of the blocks in the editor, the BlockNavigation component lets users navigate to each block by clicking on its line in the hierarchy tree.
+
+![Block navigation](https://make.wordpress.org/core/files/2020/08/block-navigation.png)
+![View of a group block navigation](https://make.wordpress.org/core/files/2020/08/view-of-group-block-navigation.png)
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+## Development guidelines
+
+### Usage
+
+Renders a block navigation with default syles.
+
+```jsx
+import { BlockNavigation } from '@wordpress/block-editor';
+
+const MyNavigation = () => <BlockNavigation
+	onSelect={ onClose }
+	__experimentalFeatures={ __experimentalFeatures }
```

Comment:
Should we document experimental things? It seems we shouldn't as they are not ready yet.

Reply:
I think so also.
We can always update this when the feature is more advanced.